ref/general/about-open-liberty.adoc

```diff
@@ -0,0 +1,80 @@
+Faster start-up times enable faster development and test cycles.
+
+=== Pluggable features
+Open Liberty features are pluggable so that your existing features are supported in new product versions. You can add, remove, and update features as needed, with minimal impact to your application. When the feature configuration is changed, Open Liberty reevaluates the code by starting and stopping parts of the runtime as necessary, without having to restart the server. For a full list of available features and their configuration elements, see link:/docs/ref/feature/#featureOverview.html[Server features].
+
+You can freely mix MicroProfile, Jakarta EE, and other features, according to the needs of your applications. For more information, see link:/docs/ref/general/#microprofile.html[MicroProfile simplifies developing cloud-native Java microservices].
+
+=== Dev mode
+Run your server in link://blog/2019/10/22/liberty-dev-mode.html[dev mode] for automatic reload and deployment, on-demand testing, and debugger support.
+With dev mode, Open Liberty automatically detects changes in your code, compiles them, and deploys them to your running server.
+You can run tests on demand, or even automatically, so that you get immediate feedback on your changes.
+
+=== Zero migration development
+With Open Liberty zero-migration architecture, you focus on writing your applications, not the APIs changing under you.
+Zero-migration architecture means that you can use existing, unmodified configuration and application files with an updated version of Open Liberty, without unexpected changes in behavior.
+Open Liberty offers full compatibility between product versions so that your existing APIs and behaviors are supported in each new release. For more information, see link:/guides/getting-started.html[Packaging and deploying applications].
+
+
+== Deploy in containers to any cloud
+
+Open Liberty can be containerized and deployed to any Kubernetes cloud. Check out these guides to learn how you can easily deploy to link:/guides/okd.html[OKD], link:/guides/cloud-openshift.html[OpenShift], link:/guides/cloud-azure.html[Azure Kubernetes Service], link:/guides/cloud-aws.html[Amazon Web Services], link:/guides/cloud-ibm.html[IBM Kubernetes Service], and more.
+
+=== Optimize throughput
+
+A properly tuned thread pool can maximize throughput, reduce response times, and increase the capacity for concurrent requests.
+Application code in Open Liberty runs in a single thread pool that is called the default executor.
+The size of this pool is set by an auto-tuning controller, which can manage a wide range of workloads.
+
+=== Zero migration deployment
+
+Zero migration is the ability to use existing, unmodified user files with an updated version of the product runtime files, with no unwanted or unexpected change in behavior. Open Liberty does not modify user configuration files, which are fully compatible between versions.
```

Comment:
"Zero migration is the ability to use existing, unmodified user files with an updated version of the product runtime files, with no unwanted or unexpected change in behavior." Here, instead of "with no unwanted or unexpected change in behavior" you could say ""without any unwanted or unexpected change in behavior